content/posts/jellyfin-10-6-0.md

```diff
@@ -0,0 +1,114 @@
+
+We're proud to announce Jellyfin 10.6's headline feature: **SyncPlay**.
+
+SyncPlay allows you to create rooms that other users or clients can join in order to share a common viewing experience. There is no limit on the number of users in a room and you are free to join the same room with the same user from multiple clients as well.
+
+Thanks to first-time contributor [OancaAndrei](https://github.com/OancaAndrei), who submitted pull requests to both the server and the web client ([jf#2733](https://github.com/jellyfin/jellyfin/pull/2733), [jf-web#1011](https://github.com/jellyfin/jellyfin-web/pull/1011)) in order to lay out the foundations for this, you can now watch movies together with friends and family, from the comfort of your respective homes.
+
+The feature is expected to be improved in future versions of Jellyfin, but has already been used by multiple users during the development cycle, with a delay between clients of only a couple of milliseconds.
+
+# Migration to Entity Framework Core
+
+It's been on our plate for a while, but thanks to new team member [barronpm](https://github.com/barronpm), we can finally say that the rewrite of our database model is progressing at a steady pace!
+
+Previously, Jellyfin used a combination of SQLite databases (yes, multiple ones), XML files and C# spaghetti to perform database operations. Information was split in multiple places, sometimes even duplicated and generally filtered in C# instead of using the database engine's faster processing.
+
+Over the course of this cycle, [barronpm](https://github.com/barronpm) has been deciphering and untangling this mess, and managed to successfully migrate the ActivityDB ([jf#2970](https://github.com/jellyfin/jellyfin/pull/2970)) and the UserDB ([jf3148](https://github.com/jellyfin/jellyfin/pull/3148)) to EF Core.
+
+While there is still a ways to go, EF Core should bring faster database queries, support for multiple database engines, cleaner code, and significantly reduced memory usage. Currently, there is still a bridge to make the link between the new EF Core databases and the existing code, which will be cleaned up down the line.
+
+Part of the improved memory usage is due to our current inherited custom ORM caching everything in memory to make up for its slowness. For large databases, this could result in hundreds of megabytes of memory lost to caching. With EF Core, however, we leave the heavy lifting to the database engine, leading to better response times and less memory usage overall.
+
+Your databases will be automatically migrated when you first launch Jellyfin 10.6. While the migration process has been well tested over the past few months, issues may arise during the migration process. To prevent any data loss, please **backup your existing data files** before starting the migration process.
```

Comment:
Can we link to a documentation page here on how to backup the data files? Not sure if one exists, but it would be nice if it did.

It might also be worthwhile to also mention this at the start of the article where the download link is provided.

> As always, please make sure you **backup your existing data files** before updating the server.